app/presenters/image_upload_response_presenter.rb

```diff
@@ -20,6 +20,16 @@ def remaining_attempts
     @form.remaining_attempts
   end
 
+  def status
+    if success
```

Comment:
> I know the method `success` is old but since it's a predicate, WDYT of renaming it `success?` to match other `success?` methods we have?

Sounds good 👍  Updated in 5e92647d0.